index.js

```diff
@@ -33,7 +33,9 @@ module.exports = function toJSON(schema) {
 
       //Set ID
       if (ret._id && typeof ret._id === 'object' && ret._id.toString) {
-        ret.id = ret._id.toString();
+        if (ret.id == null) {
```

Comment:
This check probably needs to be `if (typeof ret.id === 'undefined')`, because `null` might be a valid value